showmore_dyn/showmore_dyn.php

```diff
@@ -60,16 +57,16 @@ function showmore_dyn_settings(App &$a, &$o)
 	$t = Renderer::getMarkupTemplate('settings.tpl', 'addon/showmore_dyn/');
 	$o .= Renderer::replaceMacros($t, [
 		'$submit' => DI::l10n()->t('Save Settings'),
-		'$title' => 'Showmore Dynamic',
-		'$limitHeight' => ['limitHeight', DI::l10n()->t('Limit Height'), $limitHeight, 'The maximal height of posts when collapsed', '', '', 'number'],
+		'$header' => DI::l10n()->t('Show More Dynamic'),
+		'$limitHeight' => ['limitHeight', DI::l10n()->t('Limit Height'), $limitHeight, DI::l10n()->t('The maximal pixel height of posts before the Show More link is added, 0 to disable'), '', '', 'number'],
 	]);
 
 }
 
 function showmore_dyn_script()
 {
-	$limitHeight = DI::pConfig()->get(local_user(), 'showmore_dyn', 'limitHeight', 250);
-	$showmore_dyn_showmore_linktext = DI::l10n()->t('Show more ...');
+	$limitHeight = intval(DI::pConfig()->get(local_user(), 'showmore_dyn', 'limitHeight', 250));
+	$showmore_dyn_showmore_linktext = DI::l10n()->t('Show more...');
```

Comment:
Why have you removed the space here? I only know the German rule that you put a space when the word ended but don't use a space when the word is interrupted and I had the feeling as if that rule is some kind of universal.

Reply:
There's no space in English before any punctuation mark.

Reply:
Maybe these dots are completely superfluous.

Reply:
According to https://english.stackexchange.com/questions/26240/what-is-the-proper-way-of-using-triple-dots-and-spaces-before-after-them the space *can* be added (although most people prefer it without). So I could argue that since we are close to the RC the translators won't be able to translate the changed text :-) But in fact this doesn't seem to be translated at all at the moment.